ang/volunteer/Project.js

```diff
@@ -156,7 +156,9 @@
           "id": $scope.project.loc_block_id
         }).then(function(result) {
           if(!result.is_error) {
+            $scope.locBlockUpdating = true;
```

Comment:
I don't like the name of this flag. How about one of the following: locBlockFetch, locBlockSkipDirtyCheck?